src/main/scala/example/BorerAkkaSerializer.scala

```diff
@@ -0,0 +1,9 @@
+package example
+
+class BorerAkkaSerializer extends CborAkkaSerializer[BorerSerializable] with Codecs {
+
+  override def identifier: Int = 19923
+
+  register[Zoo]
+  register[Animal]
```

Comment:
Out of curiosity... is `register[Greeting]` not needed here? Coz there is `greetingCodec` in `Codecs` 🤔

Reply:
`register[Greeting]` is not needed, because `Greeting` is only used as a field and not a top level serializable message. Compilation would fail without `greetingCodec`.